sass/site/primary/_posts-and-pages.scss

```diff
@@ -20,10 +20,43 @@
 	display: inline;
 }
 
+.page,
+.single {
+
+	.entry-header {
+		background: $blue;
+		color: $white;
+		margin-bottom: 95px;
+		margin-left: calc(50% - 50vw);
+		max-width: none;
+		padding: 80px 40px 60px;
+		position: relative;
+		text-align: center;
+		width: 100vw;
+
+		&::after {
+			content: "";
+			background: url("#{$short_wave}") no-repeat center;
+			position: absolute;
+			left: 0;
+			bottom: -95px;
```

Comment:
Maybe you can create a sass variable for the wave height, that way it's clear where 95px comes from.